client/coral-embed-stream/src/components/Comment.js

```diff
@@ -514,19 +462,11 @@ export default class Comment extends React.Component {
                 fill="commentReactions"
                 data={this.props.data}
                 root={this.props.root}
+                asset={asset}
```

Comment:
Sure :-), but before I write an essay here, let's have a quick zoom and tag team this :-)